lib/start.js

```diff
@@ -87,7 +87,7 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
     stdio: 'inherit',
     timeout: options.network_log ? 120000 : undefined,
     continueOnFail: options.network_log ? true : false,
-    shell: true
+    shell: process.platform === 'linux' && options.network_log ? false : true
```

Comment:
shell false is better for security anyway; can we set it to false on all platforms except mac? (regardless of whether network_log is true)